### PR TITLE
oadp: Bump to go 1.19

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

**Review by sseago — APPROVED** [review]

**Review by weshayutin — APPROVED** [review]

**Comment by kaovilai** [comment]
ci/rehearse/openshift/velero/oadp-1.0/unit-test
ci/rehearse/openshift/velero/oadp-1.1/unit-test
builds fine.. just missing make-update so we can ack those.

File: ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.17
+    tag: golang-1.19
```
Comment (kaovilai) [inline ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml:5]:
```suggestion
    tag: golang-1.20
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44408/rehearse-44408-pull-ci-openshift-velero-plugin-for-csi-main-unit-test/1713989369158701056#1:build-log.txt%3A194

**Comment by kaovilai** [comment]
/pj-rehearse pull-ci-openshift-velero-plugin-for-csi-main-unit-test

**Review by weshayutin — APPROVED** [review]

**Pull request merged**